lms/static/sass/base/_layouts.scss

```diff
@@ -81,12 +85,16 @@ body.is-in-course {
 
   // site footer
   .wrapper-footer {
-    margin-top: $baseline;
+    margin-top: ($baseline*2);
     padding-right: 2%;
     padding-left: 2%;
 
-    footer#footer-openedx { // TODO check edX footer when it launches
+    footer#footer-openedx { // shame selector to match existing
       min-width: auto;
     }
   }
+
+  footer#footer-edx-v3 { // shame selector to match existing
+    margin-top: ($baseline*2);
```

Comment:
Selector above is the openedx one, and also I wanted the edx footer to always have the padding, not just on the responsive layouts. I didn't add it to the footer file because I think Alasdair matches that with the Drupal site which doesn't need the top margin. I'll add the comment though.